src/stream/consumption.ts

```diff
@@ -133,4 +134,92 @@ export class StreamConsumption<T, E> extends StreamBase {
 
         return s;
     }
+
+    /**
+     * Produce a readable node stream with the values from the stream.
+     *
+     * @param kind - What kind of readable stream to produce. When "raw" only strings and buffers can be emitted on the stream. Use "object" to preserve
```

Comment:
Not sure how (proper types in node would help), but can we make it clearer that it returns a readable stream that is operating in object mode, when applicable.